packages/web-react/src/components/Field/ValidationText.tsx

```diff
@@ -15,10 +16,14 @@ export const ValidationText = <T extends ElementType = 'div'>(props: ValidationT
   const {
     className,
     validationText,
+    validationState,
     elementType: ElementTag = defaultProps.elementType as ElementType,
     id,
     registerAria,
   } = propsWithDefaults;
+  const [role, setRole] = React.useState<string | undefined>();
```

Comment:
**polish (if-minor):** 
```suggestion
  const [role, setRole] = useState<string | undefined>();
```

Reply:
Yea, vscode doing this for me and i always forget to change it 🤦  thank you